app/backend/src/couchers/servicers/threads.py

```diff
@@ -116,6 +116,9 @@ def GetThread(self, request, context, session):
         return threads_pb2.GetThreadRes(replies=replies, next_page_token=next_page_token)
 
     def PostReply(self, request, context, session):
+        if request.content.strip() == "":
```

Comment:
Might be good to do `content = request.content.strip()`, then use `content` below on lines 124/126. That way it'll strip the surrounding whitespace from the comments/replies.